charts/osdfir-infrastructure/README.md

```diff
@@ -0,0 +1,295 @@
+| `turbinia.redis.master.count`               | Number of Redis master instances to deploy (experimental, requires additional configuration) | `1`              |
+| `turbinia.redis.master.persistence.size`    | Redis master Persistent Volume size                                                          | `8Gi`            |
+| `turbinia.redis.master.resources.limits`    | The resources limits for the Redis master containers                                         | `{}`             |
+| `turbinia.redis.master.resources.requests`  | The requested resources for the Redis master containers                                      | `{}`             |
+| `turbinia.redis.replica.replicaCount`       | Number of Redis replicas to deploy                                                           | `0`              |
+| `turbinia.redis.replica.persistence.size`   | Redis replica Persistent Volume size                                                         | `8Gi`            |
+| `turbinia.redis.replica.resources.limits`   | The resources limits for the Redis replica containers                                        | `{}`             |
+| `turbinia.redis.replica.resources.requests` | The requested resources for the Redis replica containers                                     | `{}`             |
+
+Specify each parameter using the --set key=value[,key=value] argument to helm install. For example,
+
+```console
+helm install my-release \
+    --set turbinia.enabled=false
+    oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+
+The above command installs OSDFIR Infrastructure without Turbinia deployed.
+
+Alternatively, the `values.yaml` and `values-production.yaml` file can be 
+directly updated if the Helm chart was pulled locally. For example,
+
+```console
+helm pull oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure --untar
+```
+
+Then make changes to the downloaded `values.yaml` and once done, install the 
+chart with the updated values.
+
+```console
+helm install my-release ../osdfir-infrastructure
+```
+
+## Persistence
+
+The OSDFIR Infrastructure deployment stores data at the `/mnt/osdfir` path of the container.
+
+Persistent Volume Claims are used to keep the data across deployments. This is 
+known to work in GCE and minikube. See the [Parameters](#parameters) section to configure the PVC or to disable persistence.
+
+## Upgrading
+
+If you need to upgrade an existing release to update a value, such as
+persistent volume size or upgrading to a new release, you can run 
+[helm upgrade](https://helm.sh/docs/helm/helm_upgrade/). For example, to set a 
+new release and upgrade storage capacity, run:
+```console
+helm upgrade my-release \
+    --set turbinia.server.image.tag=latest \
+    --set timesketch.image.tag=latest \
+    --set persistence.size=10T
```

Comment:
What does this do to the existing PVC, does it extend it in place without deleting anything or does it delete/re-create it?  I'm guessing the former but we should probably add a comment about that  just to be clear.

Reply:
Ack good idea, added a note around how this works but TLDR data would not be deleted/underlying volume is expanded